bakerydemo/templates/blog/blog_page.html

```diff
@@ -3,42 +3,34 @@
 
 {% block content %}
 
-    {% image self.image fill-1920x600 as hero_img %}
-        {% include "base/include/header-hero.html" %}
+{% image self.image fill-1920x600 as hero_img %}
+{% include "base/include/header-blog.html" %}
 
-    <div class="container">
-        <div class="row">
-            <div class="col-md-8">
-                {% if page.introduction %}
-                    <p class="intro">{{ page.introduction }}</p>
-                {% endif %}
-
-                <div class="blog-meta">
-                    {% if page.authors %}
-                        <div class="blog-avatars">
-                            {% for author in page.authors %}
-                                <div class="author">{% image author.image fill-50x50-c100 class="blog-avatar" %}
-                                    {{ author.first_name }} {{ author.last_name }}</div>
-                            {% endfor %}
-                        </div>
-                    {% endif %}
-
-                    {% if page.date_published %}
-                        <div class="blog-byline">
-                            {{ page.date_published }}
-                        </div>
-                    {% endif %}
+<div class="container">
+    <div class="row">
+        <div class="col-md-8">
+            <div class="blog__meta">
+                {% if page.authors %}
+                <div class="blog__avatars">
+                    {% for author in page.authors|slice:":1" %}
+                    <div class="blog__author">{% image author.image fill-50x50-c100 class="blog__avatar" %}
+                        {{ author.first_name }} {{ author.last_name }}</div>
+                    {% endfor %}
                 </div>
+                {% endif %}
+            </div>
 
-                {{ page.body }}
+            {{ page.body }}
 
-                {% if page.get_tags %}
-                    Tagged with:<br />
-                    {% for tag in page.get_tags  %}
-                        <a href="{{ tag.url }}" class="btn btn-sm">{{ tag }}</a>
-                    {% endfor %}
-                {% endif %}
+            {% if page.get_tags %}
+            <p class="blog__tag-introduction">Find more blog posts with similar tags</p>
+            <div class="blog-tags blog-tags--condensed">
+                {% for tag in page.get_tags %}
+                <a href="{{ tag.url }}" class="blog-tags__pill">{{ tag }}</a>
```

Comment:
suggestion: add context for a screenreader here to say what will happen on click